portfolio/src/main/webapp/experiment.html

```diff
@@ -0,0 +1,274 @@
+<!DOCTYPE html>
```

Comment:
Is this file based off index.html? If so, can you mark it as being copied from index.html?

Reply:
Yes it was based off index.html. The current index.html is from my last commit that has the code to how i want the main live html code to look.

Reply:
After more digging, looks like git doesn't actually have a way to mark files as copied, so nevermind on that